packages/http-client/src/ceramic-http-client.ts

```diff
@@ -154,6 +154,7 @@ export class CeramicClient implements CeramicApi {
       method: 'post',
       body: {
         queries: queriesJSON,
+        ...{ timeout },
```

Comment:
Nitpick: I don't think you need to put timeout in an object then spread
```suggestion
        timeout,
```

Reply:
I was trying to avoid sending the field with an explicit value of `undefined` over the wire